ejemplo.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>The Cheese Guardian</title>
+  <link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+  <section class="main-container">
+    <img src="phone.png" alt="foto"  />
+    <div class="vertical-separator"> </div>
+    <img src="tcg_logo.png" alt="foto"  />		
+  </section>
+  <div class="mountain-image"></div>
+  <div class="second-container"> 	
+    <div class="element">
+      <img src="ic_list.png" alt="foto"  />
+      <p class="font">Keep track of your money</p>
+      <p class="font grey">See all your debts and all your friends debts towards you.</p>
```

Comment:
no need to create a ```grey``` class, you could use a class ```feature-description``` which has the colour gray in it, this is because all the descriptions have gray colour. I think this text is a bit smaller than the headline, i would set the headlines font-size to 16px and this one to 14px for example.